CHANGES/4938.feature

```diff
@@ -0,0 +1 @@
+Add a support for Bearer token authentication
```

Comment:
s/a//
Also, can you make it clear that this support is for clients pulling from pulp? This could also be interpreted as Pulp supporting syncing from registries that use token auth, which is already in place.